shared/AppInsightsCore/src/JavaScriptSDK/AppInsightsCore.ts

```diff
@@ -32,7 +32,7 @@ export class AppInsightsCore extends BaseCore implements IAppInsightsCore {
                     if (telemetryItem === null) {
                         _notifyInvalidEvent(telemetryItem);
                         // throw error
-                        throw Error("Invalid telemetry item");
+                        throwError("Invalid telemetry item");
```

Comment:
Simply minification help as "throw" is a reserved word and "Error" is a class, so neither could be minified